app/uk/gov/hmrc/thirdpartyapplication/services/commands/grantlength/ChangeGrantLengthCommandHandler.scala

```diff
@@ -40,8 +41,17 @@ class ChangeGrantLengthCommandHandler @Inject() (
   import CommandHandler._
 
   private def validate(app: StoredApplication, cmd: ChangeGrantLength): Validated[Failures, Unit] = {
+    def printRefreshTokensAvailableFor(refreshTokensAvailableFor: Period): String = {
+      val refreshTokensAvailableForDays = refreshTokensAvailableFor.getDays
```

Comment:
Are these messages for internal API Platform use only? If they're appearing in API Gatekeeper, they will be confusing as the 180 day grant is called "6 months" there.

Reply:
These are log messages which appear in the TPA logs. GKFE shows a Grant Length changed success page for the above scenario.